backend/src/main/java/ai/verta/modeldb/entities/DatasetEntity.java

```diff
@@ -108,9 +108,6 @@ public DatasetEntity(Dataset dataset) throws InvalidProtocolBufferException {
   @Column(name = "deleted")
   private Boolean deleted = false;
 
-  @Column(name = "created")
```

Comment:
Why are we deleting this? I'm confused.

Reply:
Because now we are used RepositoryEntity for the dataset so DatasetEntity and DatasetVersionEntity are not in use. also the dataset and dataset version Dao implementation is not in use because we have replaced it with Repository and commit.

Reply:
Oh, cool. That makes sense to me